code/__DEFINES/dcs/signals/signals_mob/signals_living.dm

```diff
@@ -69,3 +69,25 @@
 // basic mob signals
 /// Called on /basic when updating its speed, from base of /mob/living/basic/update_basic_mob_varspeed(): ()
 #define POST_BASIC_MOB_UPDATE_VARSPEED "post_basic_mob_update_varspeed"
+
+/// from /datum/status_effect/incapacitating/stamcrit/on_apply()
+#define COMSIG_LIVING_ENTER_STAMCRIT "living_enter_stamcrit"
+///from /obj/structure/door/crush(): (mob/living/crushed, /obj/machinery/door/crushing_door)
+#define COMSIG_LIVING_DOORCRUSHED "living_doorcrush"
+/// Block the electrocute_act() proc from proceeding
+#define COMPONENT_LIVING_BLOCK_SHOCK (1<<0)
```

Comment:
This should be indented because its a response for shocking. This should also be placed underneath the signal that its actually returned by, which by the name probably isn't DOORCRUSHED.

Reply:
Fixed